templates/DBFile_image.ss

```diff
@@ -1 +1 @@
-<img src="$URL.ATT" alt="$Title.ATT" />
```

Comment:
There's a similar template for regular files that could be updated for this as well. Allowing regular files to have attributes as well could be interesting.

Could be a follow up issue.